tests/client/core/TestInstallAddon.js

```diff
@@ -278,6 +279,17 @@ describe('withInstallHelpers inner functions', () => {
       }));
     });
 
+    it('resets status to uninstalled on onInstallCancelled', () => {
+      const dispatch = sinon.spy();
+      const guid = '{my-addon}';
+      const handler = makeProgressHandler(dispatch, guid);
+      handler({ state: 'STATE_SOMETHING' }, { type: 'onInstallCancelled' });
+      assert(dispatch.calledWith({
```

Comment:
It's not a big deal but I don't like this style because the failure doesn't give you any information. If you switched to this style you would possibly be able to figure out the fix faster:

````js
assert.deepEqual(dispatch.firstCall.args[0], {
  type: INSTALL_CANCELLED,
  payload: { guid },
});
````